.gitignore

```diff
@@ -9,3 +9,4 @@
 storybook-static
 build-storybook.log
```

Comment:
this is good, but we must also add an example file like `.env.development.local.example` with the variables listed here with default values